package.json

```diff
@@ -82,7 +82,7 @@
     },
     "dependencies": {
         "@swc/helpers": "^0.3.13",
-        "@whereby/jslib-media": "whereby/jslib-media.git#1.3.1",
+        "@whereby/jslib-media": "whereby/jslib-media.git#havard/pan-259-make-sure-we-turn-off-camera-light-when-disabling-video",
```

Comment:
Don't forget to update this.

Reply:
Will update once the `jslib-media` PR is reviewed and merged

Reply:
Updated